devtools/datasette/fly/nginx.conf

```diff
@@ -0,0 +1,17 @@
+daemon off;
+
+events {
+  worker_connections  1024;
+}
+http {
+  server {
+    listen 8080;
```

Comment:
In theory we could turn this (and the Dockerfile) into a jinja template, which might be cleaner, but it felt like gilding the lily to me.